packages/core/src/stylable-processor.ts

```diff
@@ -1055,3 +1088,33 @@ export interface DeclStylableProps {
 export interface SDecl extends postcss.Declaration {
     stylable: DeclStylableProps;
 }
+
+export function processDeclarationFunctions(
+    decl: postcss.Declaration,
+    onFunction: (node: AnyValueNode) => void,
```

Comment:
why is the type used here (`AnyValueNode`) coming from `css-selector-tokenizer`?

Also, maybe these two utils should go in `stylable-utils.ts`?